geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSRandMemberIntegrationTest.java

```diff
@@ -64,84 +66,100 @@ public void srandmemberTooManyArgs_returnsError() {
   }
 
   @Test
-  public void srandmemberWithInvalidCount_returnsError() {
+  public void srandmember_withInvalidCount_returnsError() {
     assertThatThrownBy(() -> jedis.sendCommand(SET_KEY, Protocol.Command.SRANDMEMBER, SET_KEY, "b"))
         .hasMessageContaining(ERROR_NOT_INTEGER);
   }
 
   @Test
-  public void srandmemberWithoutCount_withNonExistentSet_returnsNull() {
+  public void srandmember_withoutCount_withNonExistentSet_returnsNull() {
     assertThat(jedis.srandmember(NON_EXISTENT_SET_KEY)).isNull();
     assertThat(jedis.exists(NON_EXISTENT_SET_KEY)).isFalse();
   }
 
   @Test
-  public void srandmemberWithCount_withNonExistentSet_returnsEmptySet() {
-    assertThat(jedis.srandmember(NON_EXISTENT_SET_KEY, 1)).isEmpty();
-    assertThat(jedis.exists(NON_EXISTENT_SET_KEY)).isFalse();
+  public void srandmember_withoutCount_withExistentSet_returnsOneMember() {
+    jedis.sadd(SET_KEY, SET_MEMBERS);
+
+    String result = jedis.srandmember(SET_KEY);
+    assertThat(result).isIn(Arrays.asList(SET_MEMBERS));
   }
 
   @Test
-  public void srandmemberWithoutCount_withExistentSet_returnsOneMember() {
+  public void srandmember_withCountAsZero_withExistentSet_returnsSubsetOfSet() {
```

Comment:
This test name is inaccurate. It seems like it should be "srandmember_withCountAsZero_withExistentSet_returnsEmptyList"